src/Tizen.NUI/src/internal/Interop/Interop.Window.cs

```diff
@@ -265,6 +265,9 @@ internal static partial class Window
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_ExcludeInputRegion")]
             public static extern void ExcludeInputRegion(global::System.Runtime.InteropServices.HandleRef window, global::System.Runtime.InteropServices.HandleRef inputRegion);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_IsWindowRotating")]
+            public static extern bool IsWindowRotating(global::System.Runtime.InteropServices.HandleRef window);
```

Comment:
could you add this here?
`            [return: global::System.Runtime.InteropServices.MarshalAs(global::System.Runtime.InteropServices.UnmanagedType.U1)]`

this is for `bool` return type.
- reference : https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/internal/Interop/Interop.Actor.cs#L55

Reply:
I see, I will update with your comment.